app/Http/Controllers/ValidateController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use LibreNMS\Validations\Database;
+use LibreNMS\Validator;
+
+class ValidateController extends Controller
+{
+    public function index(): View
+    {
+        $validator = new Validator();
+        $validator->validate();
+
+        $short_size = 10;
+
+        return view('validate.index', compact('validator', 'short_size'));
```

Comment:
I've been favoring inline multi-line array  for $data of late.  At least when things are short.

One thing that has bothered my about the validation page is that is is not ajax. :D  Not bothered if you don't implement that in this page.